include/swift/AST/Types.h

```diff
@@ -790,7 +790,10 @@ class alignas(1 << TypeAlignInBits) TypeBase {
   
   /// Check if this is a nominal type defined at the top level of the Swift module
   bool isStdlibType();
-  
+
+  /// Check if this is a CGFloat type from `CoreGraphics` module.
+  bool isCGFloatType();
+
```

Comment:
`CGFloat` also exists in open-source `Foundation`.

https://github.com/apple/swift-corelibs-foundation/blob/main/Sources/Foundation/CGFloat.swift

Reply:
Thanks! I'll need to check for both then and add some Linux tests.